backend/src/main/java/net/finman/service/EmailServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package net.finman.service;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.InputStreamSource;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Service;
+
+import net.finman.exception.EmailNotSentException;
+
+@Service
+public class EmailServiceImpl implements EmailService {
+
+    @Autowired
+    private JavaMailSender emailSender;
+
+    @Override
+    public void sendEmailWithAttachment(String to, String subject, String text, InputStreamSource attachment)
+            throws EmailNotSentException {
+        try {
+            MimeMessage message = emailSender.createMimeMessage();
+            MimeMessageHelper helper = new MimeMessageHelper(message, true);
+
+            helper.setFrom("alexIsNoob@noobmail.com");
```

Comment:
is the email sent from sonywalkmanwmfx199@gmail.com or alexIsNoob@noobmail.com?

Reply:
apparently sender overrides by the username specified in application.properties when using google's SMTP server

Reply:
so we just left this here as an easter egg

Reply:
so line 27 can be removed?

Reply:
idk didnt test

Reply:
otherwise kind of misleading

Reply:
u just dont want to be a noob